src/components/sidebar/lineView/LineHeaderTable.tsx

```diff
@@ -4,34 +4,46 @@ import _ from 'lodash';
 import classnames from 'classnames';
 import { FiInfo } from 'react-icons/fi';
 import ButtonType from '~/enums/buttonType';
-import { ILineTopic } from '~/services/lineTopicService';
+import ILineHeader from '~/models/ILineHeader';
+import SubSites from '~/routing/subSites';
+import routeBuilder from '~/routing/routeBuilder';
+import navigator from '~/routing/navigator';
 import { Button } from '~/components/controls';
 import SidebarHeader from '../SidebarHeader';
-import * as s from './lineTopicTable.scss';
+import * as s from './lineHeaderTable.scss';
 
-interface ILineTopicListProps {
-    lineTopics: ILineTopic[];
-    currentLineTopic?: ILineTopic;
+interface ILineHeaderListProps {
+    lineHeaders: ILineHeader[];
+    currentLineHeader?: ILineHeader;
     lineId: string;
 }
 
-class LineTopicTable extends React.Component<ILineTopicListProps> {
-    private redirectToEditLineTopicView = () => {
-        // TODO
-        window.alert('Toiminnon suunnittelu kesken.');
+class LineHeaderTable extends React.Component<ILineHeaderListProps> {
+    private redirectToEditLineHeaderView = (startDate: Date) => () => {
+        const editLineHeaderLink = routeBuilder
+            .to(SubSites.lineHeader)
+            .toTarget(':id', this.props.lineId)
+            .toTarget(':startDate', Moment(startDate).format())
+            .toLink();
+
+        navigator.goTo(editLineHeaderLink);
     };
-    private redirectToNewLineTopicView = () => {
-        // TODO
-        window.alert('Toiminnon suunnittelu kesken.');
+    private redirectToNewLineHeaderView = () => {
+        const newLineHeaderLink = routeBuilder
+            .to(SubSites.newLineHeader)
+            .toTarget(':id', this.props.lineId)
+            .toLink();
+
+        navigator.goTo(newLineHeaderLink);
     };
     render() {
         return (
-            <div className={s.lineTopicTableView}>
+            <div className={s.lineHeaderTableView}>
                 <SidebarHeader hideCloseButton={true}>
                     Linjan otsikot
```

Comment:
Not part of this pr but why is the title indented?

Reply:
Prettier indents it automatically. We need to change that 100 char per line restriction at some point